lib/utilities/settings_manager.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:jog_dog/utilities/settings_file_manager.dart';
+
+class Settings {
+
+  String _theme = "Dark";
+  double _tolerance = 0.05;
+
+  Settings();
+
+  Settings.fromJson(Map<String, dynamic> json) {
+    _theme = json['theme'];
+    _tolerance = json['tolerance'];
+  }
+
+  Map<String, dynamic> toJson() {
+    return {
+      'theme': _theme,
+      'tolerance': _tolerance,
+    };
+  }
+
+}
+
+class SettingsManager {
```

Comment:
Einen Settings Manager hatten wir eigentlich nicht definiert aber die Settings Klasse ist eine <<Entity>> also kann es sich selber Managen. Die Logik kann somit in die "Settings" Klasse.